org.xnec2c.Xnec2c.yml

```diff
@@ -0,0 +1,32 @@
+app-id: org.xnec2c.Xnec2c
+runtime: org.gnome.Platform
+runtime-version: '44'
```

Comment:
does it really need the GNOME sdk?

Reply:
I am not sure, however it is a GTK3 application that also depends on other GTK3 modules.

Reply:
gtk3 is in the freedesktop runtime. and reading the build isntruction it doesn't seem to require anything else

Reply:
Ah, ok. I will try switching to using a freedesktop runtime.

Reply:
Thanks!